package.json

```diff
@@ -69,7 +69,8 @@
         "build-prod": "cross-env GIT_COMMIT_HASH=$(git log -n 1 --pretty=format:'%h') craco build",
         "server": "node server/server.js",
         "dev-server": "cross-env PORT=3001 nodemon server/server.js",
-        "test": "react-scripts test"
+        "test": "react-scripts test",
+        "test-ci": "CI=true react-scripts test"
```

Comment:
Hva gjør `CI=true`? 🤔

Reply:
CI=true kjører det og returnerer 0 eller ikke 0 som exit-code. Hvis man kjører det uten CI=true så kjører testene i watch-mode for test på egen maskin.